src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -169,5 +192,5 @@ export {
   getSignOutLink,
   getBackToSignInLink,
   getSkipSetupLink,
-  getReloadPageButtonLink
```

Comment:
why `getReloadPageButtonLink` got removed? Isn't it used anywhere?

Reply:
Ah, I think I accidentally removed it when I rebasing the commits, adding it back